src/style/stem-cell-char.module.css

```diff
@@ -0,0 +1,16 @@
+.clone-column {
+    padding: 8px 0 8px 24px;
+    font-size: 12px;
+    font-weight: 700;
+}
+
+.percent-positive-column {
+    padding: 8px 24px 8px 0;
+    font-size: 12px;
+}
```

Comment:
Should the 24px padding be applied to whatever the parent is? Or could the parent do something like,
```
.parent > something > something:first-child {
  padding-left: 24px;
}

.parent > something > something:last-child {
  padding-right: 24px;
}
```